coop.v2/config.txt
1.000000e-03,1.000000e-05

code/history.txt
[2019-11-08 10:26:52.394104] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5
[2019-11-08 10:30:58.571513] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5
[2019-11-08 10:35:54.519315] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5
[2019-11-08 10:37:54.284209] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5
[2019-11-08 10:41:33.513195] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5
[2019-11-08 10:46:46.238575] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 10:48:08.895878] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 10:49:17.519568] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 10:49:51.699840] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 10:55:21.357018] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 10:55:57.567847] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 10:56:29.748414] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 10:58:13.843343] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 11:00:51.996900] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 11:01:27.706309] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:15:02.699889] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:15:55.802097] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:16:07.153221] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:17:23.273032] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:18:08.123474] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:19:35.280721] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:19:46.296815] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:24:12.963977] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:24:37.405086] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:25:46.920006] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:27:35.189377] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:28:29.025889] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:30:32.605591] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:31:09.015186] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:33:26.276855] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:34:02.734485] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:35:50.373657] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug
[2019-11-08 12:37:12.312423] python train.py --batch_size 8 --langevin_steps 30 --penalty_strength 1e-3 --step_size 0.1 --adaptive_langevin --two_stage_optim 10 --d_lr 1e-5 --debug

coop.v2/logs.bk/exp4/config.py
import tensorflow as tf

# Meta
tf.flags.DEFINE_string('name', 'exp', '')
tf.flags.DEFINE_integer('restore_epoch', -1, '')
tf.flags.DEFINE_string('restore_name', '', '')
tf.flags.DEFINE_bool('debug', False, '')
tf.flags.DEFINE_bool('viz', False, '')

# Geometry
tf.flags.DEFINE_integer('n_obj_pts', 1000, '')
tf.flags.DEFINE_integer('hand_size', 31, '')

# Model
tf.flags.DEFINE_integer('n_latent_factor', 64, '')
tf.flags.DEFINE_string('latent_factor_merge', 'concat', 'choose between "concat", "add"')
tf.flags.DEFINE_float('penetration_penalty', 0.03, '')
tf.flags.DEFINE_float('langevin_random_size', 0.1, '')
tf.flags.DEFINE_float('balancing_weight', 1e-4, '')
tf.flags.DEFINE_float('weight_decay', 1e-3, '')

# Train
tf.flags.DEFINE_integer('epochs', 10000, '')
tf.flags.DEFINE_integer('batch_size', 16, '')
tf.flags.DEFINE_float('lr_gen', 1e-3, '')
tf.flags.DEFINE_float('lr_des', 1e-3, '')
tf.flags.DEFINE_float('beta1_gen', 0.99, '')
tf.flags.DEFINE_float('beta1_des', 0.99, '')
tf.flags.DEFINE_float('beta2_gen', 0.999, '')
tf.flags.DEFINE_float('beta2_des', 0.999, '')
tf.flags.DEFINE_float('l2_reg', 0.001, '')

# Langevin
tf.flags.DEFINE_float('step_size', 0.1, '')
tf.flags.DEFINE_integer('langevin_steps', 20, '')
tf.flags.DEFINE_float('gradient_decay', 0.99, '')

flags = tf.flags.FLAGS